### PR TITLE
Feature/make efile parsing task

### PR DESCRIPTION
**Pull request by jontours**
This feature removes pandas from the API build, and adds a task to manage.py that generates json file for further parsing once the API loads.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
For testing this, we should run `./manage.py load_efile_sheets` and look at the endpoints- anything else?

**Comment by LindsayYoung** [comment]
We will also want to move pandas from requiremens to the dev requirements, right?

**Comment by jontours** [comment]
Yes, I can take care of that.  Also sorry for not following up to the previous question.  The task method `load_efile_sheets` only has to be run if any of the schema for the .csv provided by the FEC changes.  All it does is regenerate the json that is used to dereference the columns in the spreadsheet.  As is the most up to date json is provided with this PR in the `/data` directory of the API.

**Comment by LindsayYoung** [comment]
Looks like we need pandas locally for the integration tests to pass. But I think they passed before when they were just in the dev requirements.

File: manage.py [hunk 1]
```diff
@@ -296,9 +300,29 @@ def cf_startup():
         subprocess.Popen(['python', 'manage.py', 'update_schemas'])
 
 @manager.command
-def test_util():
-    df = efile_parser.get_dataframe(6)
-    efile_parser.parse_f3psummary_column_b(df)
+def load_efile_sheets():
```
Comment (LindsayYoung) [inline manage.py]: Let's add some in-line documentation and maybe a reference to this in the wiki so that people in the future will be able to find this.

**Comment by jontours** [comment]
Whoa, so I think you merged this before the tests were done.  oops.  Want me to reopen it?

**Pull request merged**